packages/wp-now/src/wp-now.ts

```diff
@@ -197,13 +197,28 @@ export default class WPNow {
 		return styleCSS.includes('Theme Name:');
 	}
 
+	static #isWpContentDirectory(projectPath: string): Boolean {
```

Comment:
Nit: There's nothing specific to `WPNow` in these private static functions, they could be regular functions decoupled from the class.

Reply:
I extracted those three to their own files.

Reply:
Lovely, thank you!